### DISCUSSION TITLE
Headless RVNC-connection to Pi5

### DISCUSSION THREAD

**Discussion by akerblom** [bareboat-necessities/lysmarine_gen#396] [Q&A] [answered]
Howdy!
It works fine to connect to my Pi with RVNC when Pi and PC is connected to home Wi-Fi. But how do I connect to the Pi from PC directly?

Tnx /A

**Comment by mgrouch** [comment, marked as answer]
Most of the users would have a router (and possibly Ethernet switch) and LAN on board. 
So just plugging Ethernet in would be enough. 

When you install BBN OS it also comes up with ‘lysmarine-hostspot’. If you connect your PC via WiFi to that hotspot then you should be able to use VNC viewer from your PC via WiFi. 

If you don’t have Ethernet or WiFi LAN there is also a way to link PC to pi via crossover Ethernet cable. Rarely someone would need that.

**Reply by akerblom** [reply]
Thank you! Just found the address under “Other ways to connect”. Think I’ll get a router but at least this works well enough 🙏
